### ISSUE TITLE
[FR] Agregar a que zona corresponde cada A/A

### ISSUE THREAD

**Issue by JMVS** [mool/home-assistant-bgh-smart#12]
Quisiera saber si es posible agregar la posibilidad de agregar a que zona corresponde cada A/A detectado. Actualmente HA arroja el siguiente error:
![imagen](https://user-images.githubusercontent.com/7751148/184366249-95f41380-27f8-4561-9013-58b31828c345.png)

Si se puede hacer a traves de la configuración yaml por favor si alguien me indica como hacerlo.
Gracias.

**Comment by Juanma04** [comment]
JMVS buenas tardes, encontraste solución a este problema? Acabo de instalarlo y tengo ese problema y además tengo dos A/A y me detecta sólo uno en las entidades, vos tenes más de un equipo y entidad? Gracias.

**Comment by JMVS** [comment]
No, no encontre solución a este problema. Tengo tres equipos y detecta los tres.

**Comment by Juanma04** [comment]
Ok gracias por tu pronta respuesta, estuve revisando un poco el código y voy a intentar corregirlo (lo del unique_id), te comento luego, slds.

**Comment by Juanma04** [comment]
Ya lo pude solucionar, mañana veo como subir el fix al repo y te aviso, feliz año nuevo!

**Comment by Juanma04** [comment]
Este es el fix que agregué en /config/custom_components/bgh_smart/climate.py

![image](https://user-images.githubusercontent.com/15800038/210176300-14df3e9b-be79-4fd1-b4c9-10bb54f4dee8.png)

Luego estaba viendo para subir un fork con eso y encontré que ya había otros 3 forks, y revisando uno de esos, encontré que ya tiene un fix similar:

https://github.com/hector-romero/home-assistant-bgh-smart

Con lo cual puedes optar por cualquiera de las dos opciones, o agregas esas líneas que puse en la imagen y reinicias o intentas cambiar el repo a ese fork. (yo no probé esto último).

Slds.
